openedx/features/pakx/lms/overrides/management/commands/create_default_partner_space.py

```diff
@@ -0,0 +1,28 @@
+"""Management command to Create A default Partner space"""
+
+from logging import getLogger
+
+from django.core.management.base import BaseCommand
+
+from openedx.features.pakx.cms.custom_settings.models import PartnerSpace
+
+log = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    A management command that create a default Partner space that will act as public space for the platform
+    """
+
+    help = "Create a default Partner space that will act as public space for the platform"
+
+    def handle(self, *args, **options):
+
+        default_space = 'ilmx'
+        from organizations.models import Organization
+        pakx_org, created = Organization.objects.get_or_create(name="ilmX", short_name=default_space, defaults={
+            'active': True
+        })
+
+        PartnerSpace.objects.get_or_create(name=default_space, organization=pakx_org)
+        log.info("Created a public space named `ilmx` linked with PakistanX org")
```

Comment:
Here organization should be `ilmX` in the logs instead of `PakistanX`

Reply:
alright